.gitignore

```diff
@@ -1,3 +1,3 @@
 go-continuous-fuzz
 
-out/
\ No newline at end of file
+test-workdir/
```

Comment:
`test-workdir` should only be present on GitHub CI workers, and IIRC CI workers start with a clean filesystem on every run.